src/main/java/frc/robot/commands/IntakeTest.java

```diff
@@ -25,11 +35,74 @@ public IntakeTest(Intake intake) {
 
   @Override
   public void initialize() {
+    intake_Motor_Encoder = Shuffleboard.getTab("intake_Test")
+    .add("intake Encoder", 0)
+    .withSize(2, 2)
+    .withPosition(0, 4)
+    .withWidget(BuiltInWidgets.kGraph)
+    .getEntry();
+    intake_Motor_Current = Shuffleboard.getTab("intake_Test")
+    .add("intake Current", 0)
+    .withSize(2, 2)
+    .withPosition(0, 4)
+    .withWidget(BuiltInWidgets.kGraph)
+    .getEntry();
+    intake_Motor_Velocity = Shuffleboard.getTab("intake_Test")
+    .add("intake Velocity", 0)
+    .withSize(2, 2)
+    .withPosition(0, 4)
+    .withWidget(BuiltInWidgets.kGraph)
+    .getEntry();
+
+    wheel_Motor_Velocity = Shuffleboard.getTab("intake_Test")
+    .add("wheel Velocity", 0)
+    .withSize(2, 2)
+    .withPosition(0, 4)
+    .withWidget(BuiltInWidgets.kGraph)
+    .getEntry();
+    wheel_Motor_Encoder = Shuffleboard.getTab("intake_Test")
+    .add("wheel Encoder", 0)
+    .withSize(2, 2)
+    .withPosition(0, 4)
+    .withWidget(BuiltInWidgets.kGraph)
+    .getEntry();
+    wheel_Motor_Current = Shuffleboard.getTab("intake_Test")
+    .add("wheel Current", 0)
+    .withSize(2, 2)
+    .withPosition(0, 4)
```

Comment:
If you look at the positions of all these entries, they will overlap, which isn't very useful. Organize them so that we can see all of them at once.